changelog/0.53.0.md

```diff
@@ -0,0 +1,95 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# sqlparser-rs 0.53.0 Changelog
+
+This release consists of 47 commits from 16 contributors. See credits at the end of this changelog for more information.
```

Comment:
> This release consists of 47 commits from 16 contributors. See credits at the end of this changelog for more information.

I think the fact we have so many contributors is directly attributable to all the work @iffyio has put into reviews. Thank you !